olive/passes/onnx/inc_quantization.py

```diff
@@ -95,6 +98,22 @@
             If you want to disable bf16 data type, you can specify excluded_precisions = ['bf16'].
         """,
     ),
+    "tuning_criterion": PassConfigParam(
+        type_=dict,
+        default_value={},
+        description="""
+            Instance of TuningCriterion class. In this class you can set strategy, strategy_kwargs,
+            timeout, max_trials and objective.
+        """,
+    ),
+    "metric": PassConfigParam(
+        type_=Metric,
```

Comment:
How about set as `type_=Optional[Metric], default_value=None`. Otherwise the type(dict) of default_value looks a bit inconsistent with `Metric`.